Header header

int16 scanType
uint16 scanIndex
uint16 targetCount
uint32 timeStamp
float32[64] p_rng
float32[64] p_rat
float32[64] p_ang
float32[64] p_amp

#TO be consistent with Vis. Copy and paste to vis/msg
